city_scrapers/spiders/chi_board_elections.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+import re
+from datetime import datetime
+
+import scrapy
+
+from city_scrapers.constants import COMMISSION
+from city_scrapers.spider import Spider
+
+
+class ChiBoardElectionsSpider(Spider):
+    name = 'chi_board_elections'
+    agency_name = 'Chicago Board of Elections'
+    timezone = 'America/Chicago'
+    allowed_domains = ['chicagoelections.com']
+    start_urls = [
+        'https://app.chicagoelections.com/pages/en/board-meetings.aspx',
+        'https://app.chicagoelections.com/pages/en/meeting-minutes-and-videos.aspx'
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows a modified
+        OCD event schema (docs/_docs/05-development.md#event-schema)
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        if response.url == \
+                "https://app.chicagoelections.com/pages/en/meeting-minutes-and-videos.aspx":
+            yield from self._prev_meetings(response)
+        if response.url == \
+                "https://app.chicagoelections.com/pages/en/board-meetings.aspx":
+            yield from self._next_meeting(response)
+
+        # self._parse_next(response) yields more responses to parse if necessary.
+        # uncomment to find a "next" url
+        # yield self._parse_next(response)
+
+    def _next_meeting(self, response):
+        next_meeting = response.xpath('//div[@class="copy"]/text()').extract()[2]
+        meetingdate = re.search(r'\d{1}:.{24}', next_meeting).group(0)
+        data = {
+            '_type': 'event',
+            'name': "Chicago Board of Election Commissioners",
+            'event_description': "Meeting",
+            'classification': COMMISSION,
+            'start': self._parse_start(meetingdate),
+            'end': {},
+            'all_day': False,
+            'location': self._parse_location(response),
+            'documents': [],
```

Comment:
The meeting notice and meeting agenda can be scraped here, and we have a good amount of examples of this in different scrapers

Reply:
I'm just going to scrape the Regular Board Meeting Agenda and not the Electoral Board Meeting Agenda or Electoral Board Meeting Notice here, because I think those two are out of date (for Sept. 27 rather than Nov 27)